dockers/docker-orchagent/tunnel_packet_handler.py

```diff
@@ -374,6 +402,8 @@ def run(self):
         Entry point for the TunnelPacketHandler class
         """
         self.wait_for_portchannels()
+        db_thread = Thread(target=self.write_count_to_db, daemon=True)
```

Comment:
Can you check if this can be as a background low priority thread?

Reply:
It doesn't seem like Python allows this level of control for threads